tests/brevitas_ort/test_quant_module.py

```diff
@@ -0,0 +1,36 @@
+from operator import mul
+from functools import reduce
+import pytest_cases
+
+import torch
+import pytest
+
+from brevitas.quant.scaled_int import Int32Bias
+from .common import *
+from pytest_cases import parametrize_with_cases
+from .quant_module_cases import QuantWBIOLCases
+from pytest_cases import parametrize_with_cases, get_case_id
+
+@parametrize_with_cases('model', cases=QuantWBIOLCases)
+@pytest.mark.parametrize('export_type', ['qop', 'qcdq'])
+def test_ort(model, export_type, current_cases):
+    cases_generator_func = current_cases['model'][1]
+    case_id = get_case_id(cases_generator_func)
+    impl = case_id.split('-')[-2] # Inverse list of definition, 'export_type' is -1, 'impl' is -2, etc.
+
+    if impl in ('QuantConvTranspose1d', 'QuantConvTranspose2d'):
```

Comment:
QuantConvTranspose is supposed to work on qcdq

Reply:
What about QOp?

Reply:
there is no quantized transposed conv in in the standard onnx opset